src/Compilers/VisualBasic/Portable/Symbols/SignatureOnlyMethodSymbol.vb

```diff
@@ -174,7 +174,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overrides ReadOnly Property IsExtensionMethod() As Boolean
             Get
-                Throw ExceptionUtilities.Unreachable
+                Return False
```

Comment:
This allows instances of this type to be displayed while debugging (GetDebuggerDisplay shouldn't throw), but we don't actually care about the value.